src/BalanceTx.purs

```diff
@@ -416,31 +427,35 @@ balanceTx unattachedTx@(UnattachedUnbalancedTx { unbalancedTx: t }) = do
         Just (Gero _) -> addTxCollateral unbalancedTx' collateral
         _ -> unbalancedTx'
 
+    availableUtxos <- lift $ map unwrap $ filterUnusedUtxos (wrap allUtxos)
```

Comment:
Minor stylistic thing: it looks a bit strange to me to use parens at the end of several `$`s